setup.py

```diff
@@ -10,16 +10,16 @@
 
 install_requires = [
     'Django>=1.8,<1.12',
-    'djangorestframework==3.6.4', # Latest version that supports both Django 1.8 and 1.11
+    'djangorestframework==3.6.4',  # Latest version that supports both Django 1.8 and 1.11
     'elasticsearch>=2.4.1,<3',
-    'requests>=2.14,<2.15',
-    'urllib3>=1.21,<1.22',
+    'requests>2.19,<2.20',
```

Comment:
I see that cfgov-refresh [is currently specifying this](https://github.com/cfpb/cfgov-refresh/blob/master/requirements/libraries.txt#L45) as `==2.18.4`, so that'd need a bump based on this change. Wagtail 1.13.x (and current master) [specifies this](https://github.com/wagtail/wagtail/blob/v1.13.4/setup.py#L34) as `>=2.11.1,<3.0`, which gives us some outer bounds.

In this project there seems to be only [this](https://github.com/cfpb/ccdb5-api/blob/aa2773f05295eef68136b19a094e86035f555bec/complaint_search/es_interface.py#L206) use of `requests` which appears to be fairly custom, so I might be a bit worried about expanding this range too much without testing/verifying that this patch still works.

Reply:
Oh yeah. Got ahead of myself here. There are several versions that need to be updated in cfgov-refresh, including `requests` and `urllib3`, and I was anticipating that change.

I'll update the range to include the version that's currently in cfgov-refresh.